README.md

```diff
@@ -174,7 +178,11 @@ Creates a new instance of the Sign-In Widget with the provided options. The widg
 - `config` - Options that are used to configure the widget
 
 ```javascript
-var signIn = new OktaSignIn({baseUrl: 'https://acme.okta.com'});
+var signIn = new OktaSignIn(
+  {
+    baseUrl: 'https://{yourOktaDomain}'
```

Comment:
Curious to know why we are changing this to `{yourOktaDomain}`. Can we add an example something like` where  {yourOktaDomain} = acme.okta.com` as a comment somewhere?

Reply:
Also I see that in developer.okta.com we say `baseUrl: https://{yourOktaDomain}.com` which is different than what we are proposing here `baseUrl: 'https://{yourOktaDomain}'`

https://developer.okta.com/code/javascript/okta_sign-in_widget

Reply:
Good questions.

We want to get all of our open source repositories to be consistent with how we are requiring the value of their Okta organization. Today, we use a mix between hardcoded example orgs like `acme.okta.com`, `acme.oktapreview.com`, etc. and `https://{yourOktaDomain}.com`. A benefit to `https://{yourOktaDomain}` is that is doesn't assume you have a production, preview, emea, etc. organization. This schema will satisfy all Okta domains - including a custom one.

Great catch! There is an [open PR](https://github.com/okta/okta.github.io/pull/1997) into D.O.C that's ready to be merged that modifies all existing `https://{yourOktaDomain}.com` over to `https://{yourOktaDomain}`.